### PR TITLE
Update em_charmm.mdp

### PR DESCRIPTION
**Pull request by VOD555**
Correct the rcoul for CHARMM forcefield.

### CODE REVIEW

**Review by orbeckst — CHANGES_REQUESTED** [review]
I don't know why but this change seems to make a test file: grompp fails for the energy minimization part. Can you please check what's going on?

**Comment by iorga** [comment]
This change is made on a file related to OPLS-AA, but the comment mentions CHARMM. @VOD555 could you please check this discrepancy? Is this change specific to OPLS or it should be also implemented in AMBER and CHARMM ? Thanks.

File: mdpow/templates/em_opls.mdp [hunk 1]
```diff
@@ -22,7 +22,7 @@ periodic_molecules       = no
 rlist                    = 1.0
 
 coulombtype              = PME
-rcoulomb                 = 1.0
+rcoulomb                 = 1.2
```
Comment (orbeckst) [inline mdpow/templates/em_opls.mdp:25]: This is the OPLS-AA file and 1.0 is correct for OPLS-AA.

**Review by orbeckst — CHANGES_REQUESTED** [review]
The OPLSAA file looks ok, do you mean the CHARMM one?

File: mdpow/templates/em_opls.mdp (hunk 1 above)
Comment (orbeckst) [reply mdpow/templates/em_opls.mdp]: Should have been in the CHARMM em MDP file. I made the changes.

**Review by orbeckst — APPROVED** [review]

**Pull request merged**